.github/workflows/audit.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     continue-on-error: true
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v5
+      uses: actions/checkout@v6
```

Comment:
Is there a reason to do this right now?  We have a monthly automated job to take care of this